src/core/server/metrics/metrics_service.ts

```diff
@@ -78,10 +78,24 @@ export class MetricsService
 
     return this.service;
   }
+  private extractOpsLogsData({ process, os }: Partial<OpsMetrics>): string {
+    const memoryLogEntryinMB = numeral(process?.memory?.heap?.used_in_bytes ?? 0).format('0.0b');
+    // ProcessMetricsCollector converts from seconds to milliseconds. Format here is HH:mm:ss for backward compatibility
+    const uptimeLogEntry = numeral((process?.uptime_in_millis ?? 0) / 1000).format('00:00:00');
+    const loadLogEntry = [...Object.values(os?.load ?? [])]
+      .map((val: number) => {
+        return numeral(val).format('0.00');
+      })
+      .join(' ');
+    const delayLogEntry = numeral(process?.event_loop_delay ?? 0).format('0.000');
+    return `memory: ${memoryLogEntryinMB} uptime: ${uptimeLogEntry} load: [${loadLogEntry}] delay: ${delayLogEntry}`;
+  }
 
   private async refreshMetrics() {
     this.logger.debug('Refreshing metrics');
     const metrics = await this.metricsCollector!.collect();
+    const opsLogsMetrics = this.extractOpsLogsData(metrics);
+    this.logger.info(opsLogsMetrics);
```

Comment:
I'm wondering if we should consider a similar approach to [what was discussed in the request/response logging issue](https://github.com/elastic/kibana/issues/13241) and log these to a new context at `debug` level so that, like the legacy platform, ops metrics are disabled with a default logging config, and you need to opt-in to get them?

Then to opt-in, you'd configure your logger like:
```yaml
  loggers:
    - context: metrics.ops
    appenders: [console]
    level: debug
```

And then [as Mikhail suggests there](https://github.com/elastic/kibana/issues/13241#issuecomment-757330953), we could:
- Explicitly create a config deprecation for the `logging.events.ops` setting
- Disable legacy ops logs when `logging.verbose: true` since you'll already be getting the new ones with that setting, so it minimizes duplicates

Reply:
Also, in addition to the backwards-compatible message string, should we consider providing these metrics in a more structured ECS-compatible way via `LogMeta`?

Reply:
I've refactored the implementation to address these suggestions. @lukeelmers would you mind taking another look please? Thanks!